integration/popular_docker_hub_images/popular_docker_images.bats

```diff
@@ -348,7 +348,7 @@ setup() {
 }
 
 @test "[create files] create files in an oraclelinux container" {
-	image="oraclelinux"
+	image="oraclelinux:8.1"
```

Comment:
For consistency, and to "surface" this hard-coded value, could you add this to:

- https://github.com/kata-containers/tests/blob/master/versions.yaml#L15

As we do for `$kibana_image`?

Reply:
@jodh-intel changes applied